src/data/faq.json

```diff
@@ -871,7 +871,7 @@
+                                    "<b>Updated on October 13, 2022:</b><div class='container'><div class='row'><div class='col-sm'><a href='/assets/img/faq/vac_booster_basics_en.png' target='_blank'><img class='img-fluid' src='/assets/img/faq/vac_booster_basics_en.png' alt='Certificate List' width='300'></a></div><div class='col-sm'><p>A booster vaccination (revaccination) is a vaccination process designed to refresh waning vaccination protection after basic immunization. In most cases, but not always, it takes place according to the recommendations of the authorities after a period of 6 months (previously 3 months) after the last vaccination procedure. (Example: completion of the vaccination series for basic immunization on October 08, 2022, possible booster vaccination on: April 08, 2023)</p><p>At <a href='https://www.zusammengegencorona.de/impfen/#id-862101356'>Vaccination Guide</a> from the Federal Ministry of Health (BMG) you will find a \"Vaccination Guide\" to your personal vaccination recommendation from the Standing Committee on Vaccination (STIKO)</p><p>Each vaccination can be attested by a digital COVID certificate, which you can scan into the Corona-Warn-App. <b>Important</b>: please ensure that the personal data (surname, first name and date of birth) are shown to match the existing vaccination certificates (and also recovery certificates, if applicable). Only then will it be possible to assign the new certificate to the other certificates (vaccinations or recovery). (Example: same academic title or indication of the full double name). All certificates of a person should be displayed grouped in the Corona-Warn-App, see figure. </p><p>Booster shots provide immediate vaccine protection.</p><p>Based on the previous vaccination series, the booster certificate will be documented '4 of 4' (in case of previous basic immunization with 3 doses).</p><p>The '2 of 1' label for a vaccination indicates that it is the first vaccination after a COVID-19 disease recovery.</p><p> See also: <a href='#vac_booster_jj' target='_blank'>Why does a 14-day wait for full vaccine protection after a booster vaccination appear in the Corona-Warn-App?</a></p></div></div></div>"

```

Comment:
<p>An interactive German-language vaccination guide from the Federal Ministry of Health (BMG) is available under <a href='https://www.zusammengegencorona.de/impfen/#id-862101356'>Impf-Guide</a> which provides a personal vaccination recommendation according to the rules of the Standing Committee on Vaccination (STIKO).</p>